### ISSUE TITLE
string extraction from react components and `getString`

### ISSUE THREAD

**Issue by hkasemir** [projectfluent/fluent.js#145]
I've been fiddling with a string extraction script to pull out localizable messages from react components that have been wrapped with `<Localized>`.

initial (and very messy) code is here: https://github.com/hkasemir/fluent-react-pull-strings

One thing I realized is that the `getString` function from the `withLocalization` HOC does not allow for any string extraction - meaning values for the strings identified by ID passed to `getString` need to be manually added to an ftl file. Is there a chance we could add a third arg to `getString` that would include a "fallback" value and to allow for optional inline defining the string for extraction scripts?

Or perhaps there's a better way to handle this, but to illustrate this idea:
```js
const name = 'Grace';
alert(getString('alert-name', {name}, 'This is an alert for { $name }');
```

**Comment by stasm** [comment]
Good idea. We could also use this third argument as the penultimate fallback—just before falling back to the `id` itself. Even without any interpolations and formatting it's still better than the `id`.

@hkasemir Would you like to submit a PR for this? The relevant code is here: https://github.com/projectfluent/fluent.js/blob/f676215a4dab988001647999d3e63e7797d8f0ab/fluent-react/src/localization.js#L65-L78
